### DISCUSSION TITLE
SimNIBS simulations using Matlab

### DISCUSSION THREAD

**Discussion by lucadirienzo** [simnibs/simnibs#237] [2. General]
Dear all, I am trying to run "tutorial_matlab.m" downloaded from the website but I obtain 

Error using run_simnibs (line 30)
There was an error running SimNIBS:
 usage: simnibs [-h] [--cpus CPUS] [--version] simnibs_file
simnibs: error: unrecognized arguments: DiRienzo\AppData\Local\Temp\tp0345a957_7e60_4620_86e6_600fdbbf2b4a.mat

Do you have any comment? Thanks

**Comment by oulap** [comment]
If you run this command in matlab:

`!echo $SIMNIBS_PYTHON`

Does it return a path?

**Comment by lucadirienzo** [comment]
Hi, it returns: $SIMNIBS_PYTHON,
so, not a path,
thanks

**Comment by diedrim** [comment]
We suggest you install the new pre-release version, please also have a look at #204.

**Reply by lucadirienzo** [reply]
Thanks, How can I install the new pre-release version?

**Reply by TorgeW** [reply]
https://github.com/simnibs/simnibs/releases/tag/dev

Download the installer and install simnibs :)

**Reply by lucadirienzo** [reply]
Thank you very much, now it works and simulations using Matlab are possible,
best regards
